src/utils/PerfUtils.js

```diff
@@ -344,6 +368,31 @@ define(function (require, exports, module) {
         return perfData[id];
     }
     
+    /**
+     * Returns the Performance metrics to be logged for health report
+     * @returns {Object} An object with the helath data logs to be send
+     */
+    function getHealthReport() {
+        var healthReport = {
+            projectLoadTimes : "",
+            fileOpenTimes : ""
+        };
+
+        _.forEach(perfData, function (entry, testName) {
+            if (StringUtils.startsWith(testName, "Application Startup")) {
+                healthReport.AppStartupTime = getValueAsString(entry);
+            } else if (StringUtils.startsWith(testName, "brackets module dependencies resolved")) {
+                healthReport.ModuleDepsResolved = getValueAsString(entry);
+            } else if (StringUtils.startsWith(testName, "Load Project")) {
+                healthReport.projectLoadTimes += ":" + getValueAsString(entry, true);
```

Comment:
Belated nit: mixing uppercase & lowercase properties names in the healthReport object seems like bad style.  Since we'll basically be stuck with those names on the storage backend once this ships, do you think that's worth cleaning up now?  CC @abose